table.csv

```diff
@@ -163,6 +163,9 @@ ed25519-priv,                   key,            0x1300,         draft,     Ed255
 secp256k1-priv,                 key,            0x1301,         draft,     Secp256k1 private key
 x25519-priv,                    key,            0x1302,         draft,     Curve25519 private key
 rsa-priv,                       key,            0x1305,         draft,     RSA private key
+p256-priv,                      key,            0x1306,         draft,     P-256 private key (compressed)
+p384-priv,                      key,            0x1307,         draft,     P-384 private key (compressed)
+p521-priv,                      key,            0x1308,         draft,     P-521 private key (compressed)
```

Comment:
```suggestion
p256-priv,                      key,            0x1306,         draft,     P-256 private key
p384-priv,                      key,            0x1307,         draft,     P-384 private key
p521-priv,                      key,            0x1308,         draft,     P-521 private key
```

I don't believe there's a distinction here for the private key form so best to remove this unless someone wants to point me to a variation where a private key can either have a compressed or uncompressed form. Other than this I'm fine with these in here.

Reply:
Done ✅